### ISSUE TITLE
Why set the block.torgb.affine of the network to be unchanged when the last random seed

### ISSUE THREAD

**Issue by jweihe** [catlab-team/stylemc#2]
`                if seed_idx == (len(seeds) - 1):
                    block.conv0.affine = torch.nn.Identity()
                    block.conv1.affine = torch.nn.Identity()
                    block.torgb.affine = torch.nn.Identity()`
What is the purpose of this operation？？？
